src/docs.js

```diff
@@ -4,44 +4,69 @@ let assert = require('assert');
 let fs = require('mz/fs');
 let path = require('path');
 let recursiveReadSync = require('recursive-readdir-sync');
+let zlib = require('zlib');
+let rootdir = require('app-root-dir');
 
 async function documenter(options) {
-
   options = _.defaults({}, options, {
-    schemas: [],
-    docsFolder: null,
+    tier: null,
+    schemas: {},
+    docsFolder: rootdir.get() + '/docs',
+    references: [],
   });
+
   assert(options.schemas, 'options.schemas must be given');
-  assert(options.schemas instanceof Array, 'options.schemas must be an array');
 
-  let schemas = options.schemas;
+  assert(options.tier, 'options.tier must be given');
+  assert(['core', 'platform'].indexOf(options.tier) !== -1, 'options.tier is either core or platform');
+
   let tarball = tar.pack();
+  let metadata = {version: 1, tier: options.tier, menuIndex: 10};
```

Comment:
Sorry, when I said default I meant that it would be set to 10 initially, but could be overridden. schemas is an example here where it defaults to `{}` but can be set to whatever you want when you instantiate it.